segmenters/nlp/Sentencizer/tests/test_sentencizer.py

```diff
@@ -57,8 +57,38 @@ def validate(req):
 
 
 def test_sentencier_cn():
+    """
+    Sentencizer does not work for chinese,
+    because string.printable does not contain Chinese characters.
+    """
     sentencizer = Sentencizer()
     text = '今天是个大晴天！安迪回来以后，我们准备去动物园。'
-    crafted_chunk_list = sentencizer.segment(text, 0)
-    # Sentencizer does not work for chinese because string.printable does not contain Chinese characters
+    crafted_chunk_list = sentencizer.segment(text)
     assert len(crafted_chunk_list) == 0
+
+
+def test_sentencier_cn_lang():
+    """
+    If we indicate the text is not in english,
+    sentencizer should be able to skip string.printable and work for chinese.
+    """
+    sentencizer = Sentencizer()
+    text = '今天又是个大晴天！安迪回来以后，我们准备再去一次动物园！'
+    crafted_chunk_list = sentencizer.segment(text, "cn")
+    assert len(crafted_chunk_list) == 2
```

Comment:
I'd merge `test_sentencier_cn ` and `test_sentencier_cn_lang `, with

```python
@pytest.fixture
def chinese_sentence():
    return '今天是个大晴天！安迪回来以后，我们准备去动物园。'

@pytest.parameteriaze(
    'lang, expected',
    [('en', 0), ['cn', 2]],
)
def test_sentencier_cn(lang, expected, chinese_sentence):
    sentencizer = Sentencizer()
    segmented = sentencizer.segment(chinese_sentence, lang)
    assert len(segmented) == expected
```

Note, the above sample illustrates the idea, not tested.

Reply:
I tried combine multiple tests together, then found I wasn't able to use the fixture in parametrize... so in latest commit instead of using pytest.fixture, I made the two sentence global variables, please kindly provide suggestions if there's better way to do that, thanks!!